src/xdocs/config.xml

```diff
@@ -861,7 +862,7 @@
 
     </section>
 
-    <section name="AuditListeners">
+    <section name="Audit Listeners">
```

Comment:
This changes the link as seen at https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/efbb9c1_2022194533/config.html#Audit_Listeners .
The link at https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/efbb9c1_2022194533/writinglisteners.html#Using_Listeners has to be updated now.

Reply:
Hey @rnveach , could you please explain this point in simple way? I couldn't get the point that you're trying to mention especially the "The link inside...".

Reply:
Every `section` tag can be turned into a link. The link created for the section tag is based on the title of the section. If you change the section title, you change the link. We link to our other sections where they make sense, as a way to say "Go look here for more information".

Reply:
Well-mentioned points! Here's a question:

How do we actually change a link here? just modify the context of the link a little bit?

Reply:
Find the similar text (for the listener in the) in the page referenced the link above (it should be named something like writinglisteners) . You'll find the `a href` there which is a link.

I assume you are familar with HTML coding?

Reply:
yeah! I'm. So Just change in the name of module in link will  ensure the change in link as well. Right?

Reply:
The only thing that needs to change is the `href` of the link. Everything can stay the same.

Reply:
@NitishKumar06 Is there any more questions or confusion? From what I was seeing, these should be the last of the changes before merging.

Reply:
@rnveach Sorry for being late! Kindly go through the changes now. I believe this PR is ready to be merged.